package/kernel/gpio-button-hotplug/src/gpio-button-hotplug.c

```diff
@@ -674,7 +674,11 @@ static void gpio_keys_irq_close(struct gpio_keys_button_dev *bdev)
 	}
 }
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6,11,0)
 static int gpio_keys_remove(struct platform_device *pdev)
+#else
+static void gpio_keys_remove(struct platform_device *pdev)
```

Comment:
I might be simpler to unconditionally switch it to void and change .remove to .remove_new.